viewer/deps.edn

```diff
@@ -0,0 +1,11 @@
+{:paths ["src"]
+ :deps {applied-science/js-interop {:mvn/version "0.3.3"}
+        binaryage/devtools {:mvn/version "1.0.3"}
+        org.babashka/sci {:mvn/version "0.5.34"
+                          #_#_#_#_:git/url "https://github.com/babashka/sci"
+                          :git/sha "a766b73f668e8ba59855ce229fad1aee08038922"}
+        reagent/reagent {:mvn/version "1.1.1"}
+        io.github.babashka/sci.configs {:git/sha "b5335b79c237344d003f16358321197a0b0a2ac6"}
+        io.github.nextjournal/viewers {:git/sha "1aaedea7709611cbb393add4c85c7a2dd551260e"}
+        metosin/reitit-frontend {:mvn/version "0.5.15"}
+        thheller/shadow-cljs {:mvn/version "2.20.7"}}}
```

Comment:
Unsure if we should add `shadow-cljs` here. 

It's not possible to use an alias from a library, correct?

Reply:
The build code we have depends directly on shadow-cljs though